parsers/SV.py

```diff
@@ -133,9 +135,9 @@ def data_processer(data) -> list:
     joined_data = sorted(d.values(), key=itemgetter("datetime"))
 
     def get_datetime(hour):
-        at = arrow.now("UTC-6").floor("hour")
-        dt = (at.replace(hour=int(hour), minute=0, second=0)).datetime
-        return dt
+        return datetime.now(TIMEZONE).replace(
+            hour=int(hour), minute=0, second=0, microsecond=0
+        )
```

Comment:
The previous version had a static offset of -6 hours from UTC, are you sure the new timezone will match this year round? Otherwise it would be better to add a static offset with datetime.timezone() instead.

Reply:
Good point, just checked! El Salvador no longer uses daylight savings (https://www.timeanddate.com/time/change/el-salvador)